index.html

```diff
@@ -345,20 +345,19 @@ <h3>
             displayItems: [
               {
                 label: "Sub-total",
-                amount: { currency: "USD", value: "55.00" },
+                amount: { currency: "GBP", value: "55.00" },
               },
               {
-                label: "Sales Tax",
-                amount: { currency: "USD", value: "5.00" },
-                type: "tax"
+                label: "Value-Added Tax (VAT)",
+                amount: { currency: "GBP", value: "5.00" },
               },
             ],
             total: {
               label: "Total due",
-              // The total is USD$65.00 here because we need to
+              // The total is GBP£65.00 here because we need to
               // add shipping (below). The selected shipping
-              // costs USD$5.00.
-              amount: { currency: "USD", value: "65.00" },
+              // costs GBP$5.00.
```

Comment:
You missed removing or replacing the `$` here

Reply:
Doh!

Reply:
Well spotted! fixed